<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAckSubstation</class>
 <widget class="QDialog" name="DlgAckSubstation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择或新建子站</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="selectRadioButton">
       <property name="text">
        <string>选择子站：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="newRadioButton">
       <property name="text">
        <string>新建子站：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="subLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>确认(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="oms_hmi.qrc">
         <normaloff>:/image/ok</normaloff>:/image/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>放弃(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="oms_hmi.qrc">
         <normaloff>:/image/cancel</normaloff>:/image/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="oms_hmi.qrc"/>
 </resources>
 <connections/>
</ui>
